studio/styles/storage.scss

```diff
@@ -58,7 +58,7 @@
 
 .dark .shimmering-loader {
   animation: shimmer 2s infinite linear;
-  background: linear-gradient(to right, #181818 4%, #333333 25%, #181818 36%);
+  background: linear-gradient(to right, #1f1f1f 4%, #333333 25%, #1f1f1f 36%);
```

Comment:
we can use css properties (variables) here instead.

like :
```css
background: linear-gradient(to right, var(--colors-scale-4)4%, var(--colors-scale-3) 25%, var(--colors-scale-4) 36%);
```

Reply:
hmm, that code probably won't work, i'll try out something locally on my machine